web/src/features/hfiCalculator/components/CalculatedPlanningAreaCells.tsx

```diff
@@ -29,6 +29,33 @@ const CalculatedPlanningAreaCells = (props: CalculatedCellsProps) => {
     daily.date.toUTC().toMillis()
   )
 
+  const calculateDailyPrepLevels = () => {
+    const prepLevels: (number | undefined)[] = []
+    range(NUM_WEEK_DAYS).map(day => {
+      const meanIntensityGroup = dailyMeanIntensityGroups[day]
+      prepLevels.push(calculatePrepLevel(meanIntensityGroup))
+    })
+    return prepLevels
+  }
+  const calculateMeanPrepLevel = (
+    rawMeanIntensityGroups: (number | undefined)[]
+  ): number | undefined => {
+    // for now, prep level calculation assumed a fixed Fire Starts value of 0-1
```

Comment:
Is this comment meant to be here?

Reply:
Yes because we are still assuming fire starts are 0-1. So that's how calculations are being done. I can take it out but it was already there before I started the ticket; I'm guessing for clarity.